sql/updates/latest-dev.sql

```diff
@@ -1,2 +1,162 @@
 DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
 DROP VIEW IF EXISTS timescaledb_information.job_stats;
+
+--Modify hypertable table.
+DROP VIEW IF EXISTS timescaledb_information.hypertables;
+DROP VIEW IF EXISTS timescaledb_information.chunks;
+
+--alter compressed column type
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable 
+ALTER COLUMN compressed DROP DEFAULT;
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+DROP CONSTRAINT hypertable_dim_compress_check;
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+DROP CONSTRAINT hypertable_compress_check;
+
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+ALTER COLUMN compressed TYPE smallint USING compressed::int;
+-- we retain the semantics for compressed column
+--retain 0 for all tables and 1 for tables that are internal
+-- compression tables.
+UPDATE _timescaledb_catalog.hypertable
+SET compressed = 2
+WHERE compressed_hypertable_id IS NOT NULL;
+
+--rename the column
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable 
+RENAME compressed TO compression_state;
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+ALTER COLUMN compression_state SET DEFAULT 0;
+
+--add the constraints
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+ADD CONSTRAINT hypertable_dim_compress_check 
+CHECK (num_dimensions > 0 OR compression_state = 1);
+
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+ADD CONSTRAINT hypertable_compress_check 
+CHECK ( (compression_state = 0 OR compression_state =2 )  OR (compression_state = 1 AND compressed_hypertable_id IS NULL));
+--recreate timescaledb_information.hypertables and chunks view 
+CREATE OR REPLACE VIEW timescaledb_information.hypertables AS
```

Comment:
It shouldn't be necessary to include the new views in the update script here. They will be recreated automatically.

Reply:
oh, ok. Will remove the view definitions.